docs/tutorials/onnx2tensorrt.md

```diff
@@ -76,11 +76,12 @@ The table below lists the models that are guaranteed to be convertable to Tensor
 |    FCOS     |   `configs/fcos/fcos_r50_caffe_fpn_4x4_1x_coco.py`   |   Y    |
 |   YOLOv3    |  `configs/yolo/yolov3_d53_mstrain-608_273e_coco.py`  |   Y    |
 |  RetinaNet  |   `configs/retinanet/retinanet_r50_fpn_1x_coco.py`   |   Y    |
-| Faster-RCNN | `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py` |   Y    |
+| Faster R-CNN | `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py` |   Y    |
 
 Notes:
 
 - *All models above are tested with Pytorch==1.6.0 and TensorRT-7.2.1.6.Ubuntu-16.04.x86_64-gnu.cuda-10.2.cudnn8.0*
+- *If the deployed model is Faster_RCNN*, please pass `max_shape = None` for the function [delta2bbox](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/bbox/coder/delta_xywh_bbox_coder.py#L134) as a workaround.
```

Comment:
Specify this workaround could done when using pytorch2onnx.py to convert models.